readme.md

```diff
@@ -40,6 +40,10 @@ $ npm -v
 
 * **Your Answer:**
 
+Cloning doesn't retain a link to the original repo.
+
+Forking retains a link to be able to merge commits back into the original repository.  The forked repo is monitored and compared with the original repo. 
+
```

Comment:
I think this is a little off. Cloning does retain a link to the original repository. Forking connects two repositories on **GitHub** whereas cloning connects a project on GitHub and a project locally.